arch/posix/Kconfig

```diff
@@ -25,6 +25,16 @@ config ARCH_DEFCONFIG
 
 source "arch/posix/core/Kconfig"
 
+config ARCH_POSIX_RECOMMENDED_STACK_SIZE
+	int
+	depends on ARCH_POSIX
+	default 24
```

Comment:
It's weird to be told about "stack size" and be shown "24, in bytes". Something got to be wrong here (e.g., it's not really *stack size* per se).

Reply:
It is the stack you allocate for a thread like for example here
https://github.com/aescolar-ot/zephyr/blob/01d0470a08a2288403932300ea072551284467d4/drivers/console/native_posix_console.c#L66

In the POSIX arch that stack is actually not used to hold the thread stack. The real stack of the thread is a normal pthread native stack.
That's why it can be so small.

Reply:
So, maybe call it DUMMY_STACK, or something. That maybe long, but at least avoid confusion (or otherwise it won't be soon when other people will understand the native port well enough ;-) ).

Reply:
That parameter will only be used for drives (or similar) meant only for the POSIX arch. So it is a bit of a hidden configuration option. I can expand the help description to clarify it though.